notebooks/02-quality-flags/TIKEBook_Lesson2.ipynb

```diff
@@ -0,0 +1,1032 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    folded_lc = lc.fold(period)
</pre><p> </p><p>Redefine lc here or up above so that it only uses the first half of data</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>done!</p>